tasks/config.yml

```diff
@@ -24,16 +25,11 @@
   notify:
     - restart td-agent
 
-- name: Remove default td-agent configuration directory
-  file:
-    path: /etc/td-agent/conf.d
-    state: absent
-
-- name: Recreate default td-agent configuration
+- name: Ensure td-agent configurations folder
   file:
     path: /etc/td-agent/conf.d
     state: directory
-    mode: 0755
+    mode: 0644
```

Comment:
mas @fadlinurhasan is this correct permission?

Reply:
`0755` gives access to execute, which is not necessary and unapplicable to a directory.
Therefore it keep getting updated

Reply:
Correct Mas. this conf doesn't need execute access.